tendenci/themes/t7-base/templates/events/badge.html

```diff
@@ -0,0 +1,165 @@
+{% load qr_code %}
+{% load base_tags %}
+<style>
+  .badge-content {
+      width: 4in;
+      line-height: 1.1;
+  }
+  .badge-section {
+      height: 3in;
+      font-size: .8em;
+      border: 1px solid black;
+  }
+  .badge-container {
+      background: #D9D9D9;
+  }
+  .badge-container .contents {
+      height: 75px;
+  }
+  .badge-section .row{
+      width: 100%;
+      margin: auto;
+      margin-bottom: 5px;
+  }
+  .badge-section .col-md-6 {
+      float: left;
+      width: 50%;
+      height: 100%;
+  }
+  .badge-section .col-md-6.info {
+      margin-top: 15px;
+      width: 100%;
+      text-align: center;
+  }
+  .badge-section .col-md-6.qr {
+      width: 38%;
+  } 
+  .badge-section .qr svg {
+      width: 39mm !important;
+      height: 39mm !important;
+      margin-top: -9px;
+  }
+  .badge-section .event_name {
+      font-size: 1.8em;
+      width: 100%;
+      text-align: center;
+      font-weight: bold;
+      margin-top: 10px;
+      height: 50px;
+  }
+  .badge-section .name {
+      font-weight: bold;
+      font-size: 1.2em;
+  }
+  .badge-section .logo {
+      width: fit-content;
+      margin-top: 5px;
+      margin-right: 7px;
+      float: right;
+  }
+  .badge-section .logo img {
+      height: 110px;
+      max-width: 200px;
+  }
+  .badge-section .registration {
+      margin-top: -10px;
+      text-align: center;
+      font-size: .8em;
+      width: 100%;
+  }
+  .badge-back {
+      font-size: .7em;
+      padding-top: 5px;
+  }
+  .badge-back .row {
+      height: 15px;
+      margin-left: 0;
+      margin-bottom: 0;
+  }
+  .badge-back .row:nth-child(odd) {
+      background-color: #D9D9D9;
+  }
+  .badge-back .col {
+      float: left;
+  }
+  .badge-back .time {
+      width: 80px;
+      margin-right: 5px;
+      margin-left: 5px;
+  }
+  .badge-back .place {
+      width: 80px;
+  }
+
+  @media print {
+      .badge-container {
+          background: #D9D9D9  !important;;
+          -webkit-print-color-adjust: exact;
+      }
+      .badge-back .row:nth-child(odd) {
+          background-color: #D9D9D9 !important;
+          -webkit-print-color-adjust: exact;
+      } 
+  }
+</style>
+
+<div class="badge-content">
+  {% if display == "front" %}
+  <div class="badge-container badge-section">
+    <div class="event_name">
+      {{ registrant.registration.event.title }}
+    </div>
+
+    <div class="contents">
+      <div class="row">
+        <div class="col-md-6 info">
+          <div class="name">
+            {{ registrant.first_name }} {{ registrant.last_name }}
+          </div>
+          <div>
+            {{ registrant.company_name }}
+          </div>
+          <div>
+            {{ registrant.pricing.title }}
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-6 qr">
+            {% qr_from_text name size="s" %}
```

Comment:
Hi Shandy,
The QR Code generated from `{% qr_from_text name size="s" %}` are all identical because the variable "name" is not defined. Based on Nicole's notes, the QR Code should be unique and linked to member/user’s registration.

Reply:
@jennyq This is a placeholder that does nothing. I think I was originally passing in the name... but changed something so it no longer does that. 
The reason, I'm using a placeholder is that the link it will point to doesn't exist yet. This is also why I added the USE_BADGES setting... so we can turn it off for everyone by default.  

The link will be added in the digital check in story. Ideally, we would have worked on that feature before this, but I got started on badges because we didn't have check in defined at that time.  (we have it defined now, but registration has to be complete before we can get started on digital check in)

Another piece that's coming soon is that currently, the back of the badge displays all child events tied to an event. In the 'add registration' work that I'm currently working on, it will be updated to only display the ones the registrant is registered for... but the full list is used for now as a placeholder.

Reply:
Gotcha. Thanks for explaining it. Since the badge code is already in the 'add-registration' branch, we can review it along with the registration update.